### PR TITLE
Fixed colors in sessions error pages

### LINKED ISSUE #8972: Check error pages in sessions styles
When there's an organization present text color is not visible with the background:

screen shot 2016-07-19 at 11 34 50 am

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #8972

screen shot 2016-07-26 at 14 08 33

CR @javierarce

### CODE REVIEW

File: app/assets/stylesheets/sessions/sessions.css.scss [hunk 1]
```diff
@@ -90,7 +97,25 @@ $cWhite: #FFFFFF;
...
+  line-height: 20px;
+  margin-bottom: 16px;
+  text-align: center;
+  color: #979EA1;
+
+  a {
+    color: $cLocationBlue;
+    text-decoration: none;
+
+    &:hover {
```
Comment (javierarce) [inline app/assets/stylesheets/sessions/sessions.css.scss]: Avoid nesting things if possible, please.

**Comment by javierarce** [comment]
Except for my comment, the rest is 👍

**Comment by xavijam** [comment]
Hey @MariaCheca, could we merge it? :)

**Comment by xavijam** [comment]
Merging due to @MariaCheca is not available today 💋 .

**Pull request merged**